l10n_br_base/tests/test_valid_createid.py

```diff
@@ -112,9 +112,9 @@ def setUp(self):
     def test_comp_valid(self):
         """Try do create id with correct CNPJ and correct Inscricao Estadual"""
         try:
-            id = self.env['res.company'].create(self.company_valid)
+            company = self.env['res.company'].create(self.company_valid)
         except:
-            assert id, u"Error when using .create() even with valid \
+            assert company, u"Error when using .create() even with valid \
```

Comment:
no more need for such unicode with Python 3

Reply:
but also, is this patch in the l10n_br_base expected in this l10n_br_zip PR?

Reply:
#667